setup.py

```diff
@@ -59,10 +59,9 @@
     "scikit-learn": ["scikit-learn>=0.22.1,<1"],
     "lightgbm": ["lightgbm>=2,<4"],
     "pytorch": [
-        "huggingface-hub>=0.0.17,<1",
-        "sentence-transformers>=2.0.0,<3",
+        "sentence-transformers>=2.1.0,<3",
         "torch>=1.9.0,<2",
-        "transformers[torch]>=4.11.0<5",
+        "transformers[torch]>=4.12.0<5",
```

Comment:
We need a comma between `4.12.0` and `<5`

Reply:
Funny, this passes linter locally 🤔